staging/prometheus-operator/patch/patch_8_prometheus_operator_crd.sh

```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env bash
-
-# This patch adds back the check for `monitoring.coreos.com/v1` that was removed from upstream.
```

Comment:
I can't remember why we needed this patch originally, but is it ok to remove the api check?

Reply:
Since we are using helm3 crds are managed in the crds dir. This no longer adds anything for us.